5/config/elasticsearch.yml

```diff
@@ -1,5 +1,5 @@
-network.host: 0.0.0.0
+http.host: 0.0.0.0
```

Comment:
This is exactly what we need! :heart: :heart: :heart:

(@yosifkit pointed this out over in https://github.com/docker-library/docs/pull/778#issuecomment-268114585, but hadn't gotten around to making a PR yet)